src/styles.ts

```diff
@@ -0,0 +1,74 @@
+import { css } from 'lit-element';
+
+export const styles = css`
+
+:host {
+    font-family: var(--font-body);
+    display: flex;
+    flex-flow: column;
+    align-items: center;
+    border: 1px solid grey;
+    padding: 12px;
+    border-radius: 12px;
+}
+h2 {
+    color: #414a51;
+    font-size: 24px;
+    text-align: left;
+}
+
+h4 {
+    text-align: center;
+}
+button {
+    font-family: var(--font-body);
+    justify-content: center;
+    margin-top: 20px;
+}
+
+.back-button {
+    padding-bottom: 20px;
+}
+
+input {
+    display: block;
+    border: 1px solid #ccc;
+    outline: none;
+    font-size: 16px;
+    font-family: 'Bariol', sans-serif;
+    font-weight: bold;
+}
+input[type="text"],
+input[type="email"],
+input[type="password"] {
+    font-size: 16px;
+    border-radius: 6px;
+    margin: 6px 0 3px;
+    width: 98%;
+    padding-left: 2%;
+    height: 32px;
+    color: #414A51;
+}
+.a {
+    color: #FF6900;
+    text-decoration: underline;
+    margin-bottom: 20px;
+}
+.link-wrapper {
+    display: flex;
+    flex-flow: column;
+    justify-content: flex-end;
+}
+.error {
+    color: red;
+    margin-bottom: 5px;
+    font-size: 14px;
+}
+
+.eye-toggle {
+    float: right;
```

Comment:
This way of positioning will error if the eye image changes size or the container gets bigger, a more reliable pattern is the following:
`
position: absolute; // and make sure the wrapper element has position: relative
right: 0; // or whatever distance from the right you need
top: 50%; // this will push your element down 50% of the height of the container it's in
transform: translateY(-50%); // this will push your element back up 50% of it's own height.
`